test/extended/workspacetype/workspace.go

```diff
@@ -61,4 +63,71 @@ var _ = g.Describe("[sig-workspace]", func() {
 			return workSpaces
 		}, 60*time.Second, 5*time.Second).ShouldNot(o.ContainSubstring(myWorkSpace.Name))
 	})
+
+	// author: zxiao@redhat.com
+	g.It("Author:zxiao-Medium-[Smoke] I can list child workspaces under parent workspace", func() {
+		g.By("# Create a test workspace as parent workspace")
+		k.SetupWorkSpace()
+		parentWorkSpace := k.WorkSpace()
+
+		g.By("# Create five child workspaces under the parent workspace")
+		childWorkSpaces := []exutil.WorkSpace{}
+		for i := 1; i <= 5; i++ {
+			g.By(fmt.Sprintf("# Creating child workspace no.%v", i))
+			k.SetupWorkSpaceWithSpecificPath(parentWorkSpace.ServerURL)
+			childWorkSpace := k.WorkSpace()
+			childWorkSpaces = append(childWorkSpaces, childWorkSpace)
+		}
+
+		g.By("# List child workspaces under the parent workspace, check if all workspaces are correctly displayed")
+		workSpaces := k.ListWorkSpacesWithSpecificPath(parentWorkSpace.ServerURL)
+		for i := 0; i < len(childWorkSpaces); i++ {
+			childWorkSpace := childWorkSpaces[i]
+			// NOTE: WorkSpace name inequality bug: https://url.corp.redhat.com/1878a60
+			var workSpace string
+			for j := 0; j < len(workSpaces); j++ {
+				workSpace = workSpaces[j]
+				if strings.Contains(workSpace, childWorkSpace.Name) {
+					break
+				}
+			}
+			o.Expect(workSpaces).To(o.ContainSubstring(childWorkSpace.Name))
```

Comment:
How about just run kubectl get workspaces childWorkSpaces to test your checkpoints?